go/protocol/keybase_v1.go

```diff
@@ -367,7 +364,7 @@ const (
 	InstallStatus_INSTALLED     InstallStatus = 4
 )
 
-type ServiceStatusError struct {
+type StatusError struct {
```

Comment:
Is it possible to use our regular `Status` field here? We're using that elsewhere to move Go Errors over msgpack RPC.